lib/matplotlib/colors.py

```diff
@@ -960,6 +960,229 @@ def scaled(self):
         return (self.vmin is not None and self.vmax is not None)
 
 
+class FuncNorm(Normalize):
+    """
+    Creates a normalizer using a custom function
+
+    The normalizer will be a function mapping the data values into colormap
```

Comment:
I forgot to address this in my previous commit. What about saying: 

 The normalizer will use a provided custom function to map the data values into colormap values in the [0,1] range.